build.sh

```diff
@@ -108,6 +113,7 @@ boot () {
 }
 
 image () {
+	mkdir "${out}" || return 1
```

Comment:
I think we don't need another variable here. `dirname "${image}"` can give you the name of the directory. It should also use `mkdir -p` instead of `mkdir`, or it will fail when the parent directory doesn't exist or the target directory already exists.